zubbi/doc.py

```diff
@@ -98,16 +107,18 @@ def render_sphinx(content):
 
         # Extract the platforms from the logger output
         platforms = []
+        reusable = False
         status_log.seek(0)
         for line in status_log.readlines():
-            prefix, _, platform = line.partition(":")
-            if prefix == SUPPORTED_OS_LOG_PREFIX:
-                platforms.append(platform.strip().lower())
-
+            prefix, _, value = line.partition(":")
+            if prefix == SUPPORTED_OS_DIRECTIVE_NAME:
```

Comment:
Here we could use the class attribute as well, so we can specify the value directly in the `directive_name` and don't need an additional constant.

Reply:
will fix.